backend/src/main/java/com/redhat/cloud/notifications/routers/ValidationEndpoint.java

```diff
@@ -20,7 +21,12 @@
     public Uni<Response> validate(@RestQuery String bundle, @RestQuery String application, @RestQuery String eventType) {
         return appResources.getEventType(bundle, application, eventType)
                 .onItem()
-                .transform(this::isValid);
+                .transform(this::isValid)
+                .onFailure(NoResultException.class).recoverWithItem(this::convertToResponse);
```

Comment:
This still doesn't work as expected. It probably reacts on failures in the previous transform method instead of the getEventType() call.. I don't know how to fix it. Right now it returns a http 500 instead of a 404 in case an NoResultException appears.. can someone help me fixing it?